Model/GridExportType/AbstractExportType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Hyva\Admin\Model\GridExportType;
+
+use Hyva\Admin\Model\ExportInterface;
+use Hyva\Admin\ViewModel\HyvaGrid\ColumnDefinitionInterface;
+use Hyva\Admin\ViewModel\HyvaGridInterface;
+use Magento\Framework\App\Filesystem\DirectoryList;
+
+abstract class AbstractExportType implements ExportInterface
+{
+    private HyvaGridInterface $grid;
+    private string $fileName;
+    private string $contentType = 'application/octet-stream';
+
+    public function __construct(HyvaGridInterface $grid, string $fileName) {
+        $this->grid = $grid;
+        if ($fileName) {
```

Comment:
This prevents having an empty filename. Instead of that it should then stick to the default-filename provided in the exportType (e.g. stay export.csv when passing an empty string which is not quite nice to download or to name a file).